pages/pipelines/trigger_step.md.erb

```diff
@@ -9,6 +9,12 @@ steps:
   - trigger: "another-pipeline"
 ```
 
+The author of the initial build will be the author of all builds created from a trigger step. 
+
+If you have [Teams](/docs/pipelines/permissions) enabled in your organisation, ensure that this author has Build permission on all of the pipelines that will be triggered by this step.
+
+If the author does not have the required access, the initial build will fail, and builds on subsequent pipelines will not be triggered.
```

Comment:
This is good, but it’s missing the requirement that the author be part of the org!

Maybe something like

>Any builds created by a trigger step will have the same author as the initiating build.
>
>The author of the build containing the trigger step must be a member of your organisation with a verified email address in order to trigger further builds. If you have [Teams](/docs/pipelines/permissions) enabled in your organisation, the author must have Build permission on the pipelines that will be triggered by this step.
>
>If the author does not have the required access, the trigger step will fail, and the requested builds are not created.